### PR TITLE
Full Coverage

### PR DESCRIPTION
**Pull request by bh2smith**
Beefing-up our unit tests to achieve full coverage.

**TestPlan:**

Run `npm run coverage` and see the 100% in all places!

### CODE REVIEW

File: contracts/SnappAuction.sol [hunk 1]
```diff
@@ -109,7 +109,7 @@ contract SnappAuction is SnappBase {
 
         // Update Auction Hash based on request
         uint16 accountId = publicKeyToAccountMap(msg.sender);
-        require(accountId <= AUCTION_RESERVED_ACCOUNTS, "Accout is not a reserved account");
+        require(accountId <= AUCTION_RESERVED_ACCOUNTS, "Account is not a reserved account");
```
Comment (bh2smith) [inline contracts/SnappAuction.sol]:
```suggestion
        require(fart <= AUCTION_RESERVED_ACCOUNTS, "Account is not a reserved account");
```

**Review by josojo — APPROVED** [review]
Neat! fully covered coverage!

File: test/snapp_auction.js [hunk 2]
```diff
@@ -169,13 +177,35 @@ contract("SnappAuction", async (accounts) => {
...
       )
     })
 
+    it("Rejects standing order requests from non-reserved accounts", async () => {
+      const instance = await SnappAuction.new()
+
+      const numReservedAccounts = (await instance.AUCTION_RESERVED_ACCOUNTS()).toNumber()
+
+      const tokens = await registerTokens(MintableERC20, instance, token_owner, 1)
+      const amount = "300000000000000000000"
```
Comment (fleupold) [inline test/snapp_auction.js]: ubernit: Do we have to fund the account? Also 1 one order would probably be enough. (Just trying to reduce the amount of action in this test to the necessary minimum).

**Review by fleupold — APPROVED** [review]
👮

File: test/snapp_auction.js (hunk 2 above)
Comment (bh2smith) [reply test/snapp_auction.js]: I'll take a look. It is possible we don't need to fund the account.

**Pull request merged**